src/art/art.c

```diff
@@ -74,6 +77,8 @@ typedef struct art_node16_s {
 typedef struct art_node48_s {
     art_inner_node_t base;
     uint8_t count;
+    // Bitset where the ith bit is set if children[i] is available
+    uint64_t available_children;
```

Comment:
Now that we have this mask, we could use `64 - popcount` to determine the number of children, which would remove the need to keep `count` around.

Reply:
I thought about doing this, but it doesn't save any size because of padding (even if we tried `uint8_t available_children[6]` to use 48 bits exactly), and I expect it's faster to keep the count separately, but I haven't benchmarked.